ll.tm

```diff
@@ -2018,6 +2019,7 @@ ValueInstruction -> ValueInstruction
 	| LandingPadInst
 	| CatchPadInst
 	| CleanupPadInst
+	| FreezeInst
```

Comment:
The list order is based on the LLVM LangRef order for instructions. As such, we can add `FreezeInst` between `SelectInst` and `CallInst`.

**Edit:** I'll merge this PR and send a follow-up commit to update the order, and perhaps add a comment about the order used.

![screenshot_2020-04-14_16:34:55](https://user-images.githubusercontent.com/1414531/79237192-2e821200-7e6e-11ea-9565-fbb8fd83ce6b.png)